### ISSUE TITLE
SBML import breaks on MATHML `power`

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/AMICI#605]
To reproduce run amici_import_petab.py on https://github.com/LoosC/Benchmark-Models/tree/hackathon/hackathon_contributions_new_data_format/Sneyd_PNAS2002 which results in:
```
model_Sneyd_PNAS2002_dydx.cpp:16:41: error: ‘_xi_1’ was not declared in this scope
 0.10000000000000001*Subs(Derivative(pow(_xi_1, 4), _xi_1), (_xi_1,), (0.9*IPR_A + 0.1*IPR_O,));
                                         ^~~~~
```
and the like

**Comment by FFroehlich** [comment]
Might me possible this be fixed by using `sbml.formulaToL3String(x.getMath())` instead of `x.getFormula()` in l687 of `sbml_import.py`

**Comment by dweindl** [comment]
Thanks for the suggestion. Doesn't, fix it, but still changing it.

Will need to replace the power in the string or teach sympy to handle the power function. Not on my priority list.

**Comment by paulstapor** [comment]
Interestingly, that's actually a bug in PEtab.
Fixed in ICB-DCM/PEtab#87
